boards/arm/gd32f4/gd32f450zk-eval/include/board.h

```diff
@@ -28,7 +28,7 @@
 #include <nuttx/config.h>
 
 #ifndef __ASSEMBLY__
-#  include <stdint.h>
+#include <stdint.h>
```

Comment:
revert the change

Reply:
@GD32-MCU but this chagne doesn't revert yet.

Reply:
Thanks for your  review,  now the chagne is revert